community-nightlies/deb.sh

```diff
@@ -198,7 +204,7 @@ main ()
     echo "If you are running a supported OS, please contact us via https://www.citusdata.com/about/contact_us and report this."
     [ -e $apt_source_path ] && rm $apt_source_path
     exit 1
-  elif [ "$curl_exit_code" = "35" ]; then
+  elif [ "$curl_exit_code" = "35" -o "$curl_exit_code" = "60" ]; then
```

Comment:
The shellcheck linter suggests:

[SC2166](https://github.com/koalaman/shellcheck/wiki/SC2166): Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.

Also there are several [libcurl error codes](https://curl.haxx.se/libcurl/c/libcurl-errors.html) that have to do with cert stuff. I don't know enough about it to know which codes are relevant here, but curious about your motivation to add this particular one.

Reply:
All of the changes here came from the upstream packagecloud `deb.sh` and `rpm.sh` scripts, including the curl exit code checks. I don't know what's causing them to check for it, but I've been keeping our script as a superset of theirs so I just merged their changes into our scripts.